flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/logical/operators.scala

```diff
@@ -215,14 +215,22 @@ case class Aggregate(
   }
 
   override def validate(tableEnv: TableEnvironment): LogicalNode = {
-
+    implicit val relBuilder: RelBuilder = tableEnv.getRelBuilder
     val resolvedAggregate = super.validate(tableEnv).asInstanceOf[Aggregate]
     val groupingExprs = resolvedAggregate.groupingExpressions
     val aggregateExprs = resolvedAggregate.aggregateExpressions
     aggregateExprs.foreach(validateAggregateExpression)
     groupingExprs.foreach(validateGroupingExpression)
 
     def validateAggregateExpression(expr: Expression): Unit = expr match {
+      case Alias(child, _, _) => validateAggregateExpression(child)
+        // check user-defined aggregate function
+      case AggFunctionCall(agg, _) if agg.requiresOver =>
```

Comment:
Check this case also as 
```
case aggExpr: Aggregation if aggExpr.getSqlAggFunction.requiresOver =>
```

in case we add built-in Aggregations that require Over.